iroh/src/fixture.rs

```diff
@@ -51,6 +53,26 @@ fn fixture_get() -> MockApi {
     api
 }
 
+fn fixture_add_file() -> MockApi {
+    let mut api = MockApi::default();
+    api.expect_add_file().returning(|_ipfs_path, _| {
+        Box::pin(future::ready(
+            Cid::from_str("QmYbcW4tXLXHWw753boCK8Y7uxLu5abXjyYizhLznq9PUR").map_err(|e| e.into()),
```

Comment:
So this returns the same cid whatever you add. Is the intention to make it more realistic, or just write another mock for that?

Reply:
Just write another mock if that becomes a use case. In general I don't think we should strive for realism in these mocks, just fulfilling the contract in the easiest way possible for a particular test case.

Unfortunately this means for `add` there isn't all that much to mock yet. Perhaps that will change somewhat once the `add` API starts to return more stuff related to progress, though I suspect tests for that will have to be normal Rust unit tests, not `trycmd` tests.